index.html

```diff
@@ -262,14 +263,19 @@
           <li>Remove a listener from an event.</li>
         </ul>
       </li>
+      <li>Retrieve a thing that is exposed by the local runtime</li>
       <li>Create and expose a local <a>Thing</a> based on a <a>Thing Description</a>.</li>
       <li>Programmatically create and expose a local <a>Thing</a>. This may include the following operations:
         <ul>
           <li>Add a property definition to the <a>Thing</a>.</li>
           <li>Add an event definition to the <a>Thing</a>.</li>
           <li>Add an action definition to the <a>Thing</a>.</li>
+          <li>Attach semanic information to an Action</li>
+          <li>Attach semanic information to a Property</li>
+          <li>Attach semanic information to an Event</li>
           <li>Start the exposed <a>Thing</a> in order to process external requests.</li>
-          <li>Handle an external request to retrieve the <a>Thing Description</a>.</li>
+          <li>Modify the handling of an external request to retrieve the <a>Thing Description</a>.</li>
```

Comment:
some reformulation would be needed; does it mean a custom handler for the external request to retrieve the TD, or a piece of code that can act upon the Thing and TD before the implementation generates the response?